src/core/object/controller/controller.ts

```diff
@@ -902,7 +958,7 @@ export default class Controller {
 				void this.setSongNowPlaying();
 			} else {
 				// Resend current mode
-				this.setMode(this.mode);
+				this.setMode(this.getMode());
```

Comment:
```suggestion
				this.onModeChanged();
```
To avoid mutating this.mode

Reply:
Good catch, thanks for catching it 🥲 https://github.com/web-scrobbler/web-scrobbler/pull/4472/commits/6fdfbcf6206d1fe8a6d0afdc21a4d2611d9158bc